source/io/module_javaScriptInvoke.js

```diff
@@ -432,18 +445,36 @@
                     throw new Error('The API being accessed for ' + functionName + ' is not valid anymore.');
                 }
                 completionCallbackStatus.retrievalState = completionCallbackRetrievalEnum.RETRIEVED;
-                return generateCompletionCallback(occurrencePointer, functionName, returnTypeName, returnValuePointer, errorStatusPointer, errorCodePointer, errorSourcePointer, completionCallbackStatus);
+                return generateCompletionCallback(occurrencePointer, functionName, returnTypeName, returnValuePointer, errorStatusPointer, errorCodePointer, errorSourcePointer, completionCallbackStatus, isInternalFunction);
             };
+
+            if (isInternalFunction) {
+                api.setLabVIEWError = function (status, code, source) {
+                    Module.coreHelpers.mergeErrors(status, code, source, errorStatusPointer, errorCodePointer, errorSourcePointer);
+                };
+            }
             return api;
         };
 
+        publicAPI.javaScriptInvoke.registerInternalFunctions = function (functionsToAdd) {
+            Object.keys(functionsToAdd).forEach(function (name) {
+                if (internalFunctionsMap.has(name)) {
```

Comment:
Can we add one more assert please? :)

typeof name === 'string' && name.length > 0